content/docs/iac/concepts/options/ignorechanges.md

```diff
@@ -93,19 +107,319 @@ resources:
 
 {{< /chooser >}}
 
-One reason you would use the `ignoreChanges` option is to ignore changes in properties that lead to diffs. Another reason is to change the defaults for a property without forcing all existing deployed stacks to update or replace the affected resource. This is common after you’ve imported existing infrastructure provisioned by another method into Pulumi. In these cases, there may be historical drift that you’d prefer to retain, rather than replacing and reconstructing critical parts of your infrastructure.
+One reason you would use the `ignoreChanges` option is to ignore changes in properties that lead to diffs. Another reason is to change the defaults for a property without forcing all existing deployed stacks to update or replace the affected resource. This commonly occurs after importing existing infrastructure provisioned by another method into Pulumi. In these cases, there may be historical drift that you’d prefer to retain, rather than replacing and reconstructing critical parts of your infrastructure.
+
+## Example: Preserve externally managed weights
```

Comment:
It's kinda funny to list 2 reasons why you might want to use `ignoreChanges`, then give an example for a _third another reason:

- ignore properties that lead to diffs
- change defaults without effecting existing resources

The example case should probably be listed in the set of resources you might use `ignoreChanges`.

Reply:
I updated this to be a list instead of a single paragraph. Let me know what you think.

Reply:
Looks great.